src/NuGet.Core/NuGet.Frameworks/NuGetFramework.cs

```diff
@@ -334,6 +348,14 @@ public bool IsSpecificFramework
             get { return !IsAgnostic && !IsAny && !IsUnsupported; }
         }
 
+        /// <summary>
+        /// True if this framework is Net5 or later, until we invent something new.
+        /// </summary>
+        public bool IsNet5Era
```

Comment:
This really, really feels like an internal implementation detail that shouldn't be public.

Also, I'm not a fan of the name. The comment suggests that in the future we're going to invent something new. Are we going to create a new boolean for each "era"?  How about using an enum instead?

Reply:
And it won't need to be once the warning has been removed.

Reply:
yes, agree...i over publicized.
going to be internal, since other classes around need this.
since it is internal now...going to keep a bool. once the next era comes, we can decide how to combine this into an enum or not.
next commit has this fixed.